pom.xml

```diff
@@ -47,9 +47,14 @@
     <scala.binary.version>2.12</scala.binary.version>
     <scala.version>2.12.15</scala.version>
     <spark.major.version>3</spark.major.version>
-    <sparkbundle.version>3.2</sparkbundle.version>
+    <sparkbundle.version>3.4</sparkbundle.version>
```

Comment:
Does it mean when no `-Pspark-<version>` specified, Spark 3.4 will be used by default? So the change includes a default version bump (from 3.2 to 3.4) for Spark?

Reply:
@zhztheplayer Thanks for your review.  Yes, default Spark version will be changed.
PR #3727 already changed `spark.version` to 3.4.1 because reported security issue by local code scan. 
So I think it's better to make `sparkbundle.version` consistent with `spark.version`.